### DISCUSSION TITLE
GrandeModular update

### DISCUSSION THREAD

**Discussion by dbgrande** [DISTRHO/Cardinal#478] [General]
I have just released a new version of GrandeModular (tag = 2.7.2), which fixes an important bug in my VCA4 module.

Cardinal is currently using the 2.5.0 release.  There have been 14 commits, 6 releases to the VCV Rack library, and two new modules: Compare3 and Tails4, since that version.

repo: [GrandeModular](https://github.com/dbgrande/GrandeModular)
branch: v2
commit: bc7ef8157f6ad1284d7c4706eb89132f59555ca1
tag: 2.7.2

**Comment by dromer** [comment]
Nice, you can also create a pull request that updates the submodule and adds the new modules ;)

I'll see if I can do this soon-ish.

**Reply by dromer** [reply]
https://github.com/DISTRHO/Cardinal/pull/479

Will probably go into the 23.01 release :)

**Comment by dbgrande** [comment]
I'm guessing you'll probably want to update to my latest version of GrandeModular (tag = 2.8.0).

This one adds the option to use dark panels for all my modules, which should give you a more proper dark mode (expletive deleted).

I'm using Rack 2.4's new "Use dark panels if available" view option to switch panels.

same repo as above
commit: 2f5f9d84a3c1c9500c0bbd4bcc284dfe0d15209d
tag: 2.8.0

**Comment by dbgrande** [comment]
I re-downloaded the latest Cardinal source and tried to update GrandeModular, but Cardinal doesn't yet support Rack 2.4's additions for "Use dark panels if available."

Here's what GrandeModular dark mode looks like:

![Grande-dark_2 8 0](https://github.com/DISTRHO/Cardinal/assets/64602967/e3666a28-77dd-4867-9f2e-a31499246b6d)

**Reply by falkTX** [reply]
for the time being cardinal stays on the 2.3.x rack series, as the dark prefer mode from vcv rack is still not in full swing, lots of modules not yet properly updated to it.

but if you send a pr for the updates, I can fix up the cardinal side to add the new global bool var following cardinal's already existing dark mode.

that dark mode looks great btw

**Comment by dbgrande** [comment]
Just noticed you've started adding support for 2.4 and dark mode.  The latest GrandeModular version now compiles into Cardinal just fine, and the dark panels option works great.

Hopefully, you can update your version of the GrandeModular plugin to 2.8.0 before your next official release.

**Reply by falkTX** [reply]
Indeed we can do that now, updated in d747d918bb8e95a8726c57dce68c117bee2fe764
